### PR TITLE
build: enforce consistent readonly array type

### PR DESCRIPTION
**Pull request by rafaelss95**
Relates to:

- #19640
- https://github.com/angular/angular/issues/33135
- https://github.com/angular/components/pull/20252#discussion_r467832026

This PR brings normalization on the form of write readonly arrays: `readonly TYPE []`.

Note that for some cases, specifically truly constants, I've opted out to use `as const` as it's shorter and fully immutable without the need of declaring the type.

### CODE REVIEW

File: src/cdk/drag-drop/parent-position-tracker.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@
  */
 
 import {ViewportRuler} from '@angular/cdk/scrolling';
-import {getMutableClientRect, adjustClientRect} from './client-rect';
+import {adjustClientRect, getMutableClientRect} from './client-rect';
```
Comment (crisbeto) [inline src/cdk/drag-drop/parent-position-tracker.ts:10]: Note that reordering the imports will make it harder to land and will cause conflicts with existing PRs.

Comment (rafaelss95) [reply src/cdk/drag-drop/parent-position-tracker.ts:10]: > Note that reordering the imports will make it harder to land and will cause conflicts with existing PRs.

That's my IDE config configuration, it always sorts it alphabetically :)
Anyway, if it causes too much noise, I can disable this configuration and rollback it.

Comment (crisbeto) [reply src/cdk/drag-drop/parent-position-tracker.ts:10]: It might be best to roll it back since it causes a lot of churn for existing PRs.

**Comment by annieyw** [comment]
@andrewseguin

**Review by andrewseguin — APPROVED** [review]

**Pull request merged**